src/titiler/core/titiler/core/dependencies.py

```diff
@@ -468,3 +471,17 @@ def __post_init__(self):
 
         if self.range:
             self.range = list(map(float, self.range.split(",")))  # type: ignore
+
+
+def PostProcessParams(
+    algorithm: algos.names = Query(None, description="Algorithm name", alias="algo"),
+    algorithm_params: str = Query(
+        None, description="Algorithm parameter", alias="algo_params"
+    ),
+) -> Optional[BaseAlgorithm]:
+    """Data Post-Processing options."""
+    kwargs = json.loads(algorithm_params) if algorithm_params else {}
+    if algorithm:
+        return algos.get(algorithm.name)(**kwargs)
+
+    return None
```

Comment:
because `titiler.core.algorithm.algos` is `frozen` people registering their own algorithm will need to re-write the `PostProcessParams` dependency.

Let's find a way not to do this